dbt/models/eia923/_core_eia923__yearly_byproduct_disposition/schema.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+sources:
+  - name: pudl
+    tables:
+      - name: _core_eia923__yearly_byproduct_disposition
+        data_tests:
+          - expect_columns_not_all_null
+          - check_row_counts_per_partition:
+              table_name: _core_eia923__yearly_byproduct_disposition
+              partition_column: report_year
+          - expect_sum_close_to_total_column:
```

Comment:
This sums across the columns, right? So it's always consistent in units (a steam record is all mmbtu, a coal ash record is always tons, etc.)